public/views/walletHome.html

```diff
@@ -219,7 +216,16 @@ <h4 class="title m0">
          </div>
         </div>
 
-        <div ng-repeat="btx in index.txHistory"
+        <div id="searchLabel" class="left-inner-addon ">
+          <i class="fi-magnifying-glass"></i>
+          <input ng-click="home.searchInput()"
+                ng-focus="home.formFocus(true)"
```

Comment:
The value that receives `formFocus` isn't boolean. It 's an string with the names of the input.